src/Extensions/Array.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe } from 'riteway'
+import './Array'
+
+describe('Array', async (should: any) => {
+  const { assert } = should('')
+
+  {
+    const arrayWithObject = [
+      ['key1', 'value1'],
+      ['key2', 'value2']
+    ]
+
+    const result = arrayWithObject.toObject()
+
+    assert({
+      given: 'array with two pairs of key values',
+      should: 'transform to an object',
+      actual: typeof result === 'object',
+      expected: true
+    })
+
+    assert({
+      given: 'array with two pairs of key values',
+      should: 'get the keys',
+      actual: Object.keys(result),
+      expected: ['key1', 'key2']
+    })
+
+    assert({
+      given: 'array with two pairs of key values',
+      should: 'get the values',
+      actual: Object["values"](result),
```

Comment:
Object.values(result) should work here. And it will be consistent with Object.keys() above.

Reply:
yes, you're right, but Typescript doesn't like. I don't know why my IDE shows me this error.

`ts] Property 'values' does not exist on type 'ObjectConstructor'.`

But I will change it, If it pass all CI I think there will not be any problem